core/math/math_funcs.h

```diff
@@ -199,6 +200,10 @@ class Math {
 		return value;
 	}
 	static _ALWAYS_INLINE_ int posmod(int p_x, int p_y) {
+		if (p_y == 0) {
+			ERR_FAIL_V_MSG(-1, "p_y is 0");
```

Comment:
Perhaps this could be more descriptive?
```suggestion
			ERR_FAIL_V_MSG(-1, "Cannot divide by zero.");
```